TrackingTools/TrackRefitter/interface/TrackTransformer.h

```diff
@@ -46,7 +46,16 @@ class TrackTransformer final : public TrackTransformerBase{
   ~TrackTransformer() override;
 
   /// fillDescriptions
-  static void fillPSetDescriptions(edm::ParameterSetDescription& descriptions);
+  static void fillPSetDescription(edm::ParameterSetDescription& descriptions,
+				  bool DoPredictionsOnly=false,
+				  const std::string& Fitter="KFFitterForRefitInsideOut",
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
@slava77 all other arguments here are uppercase, this addition seems consistent. Do you suggest modifying the whole interface?

Reply:
my comment implied changing all.